app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+<div class="cat-img-big"><%= image_tag(@booking.cat.picture_url) %></div>
+  <p><b>Name: </b><%= @booking.cat.name %></p>
+  <p><b>Breed: </b><%= @booking.cat.breed %></p>
+  <p><b>Color: </b><%= @booking.cat.color %></p>
```

Comment:
please delete breed + color and add location

Reply:
Done